YellowNotes.Api/Filters/AuthorizeActionFilter.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Mvc.Authorization;
+
+namespace YellowNotes.Api.Filters
+{
+    public class AuthorizeActionFilter : AuthorizeFilter
```

Comment:
to be removed, it's empty, we don't want to maintain empty, redundant code

Reply:
This was made to fix #48. There is no other way to register [this filter](https://docs.microsoft.com/pl-pl/dotnet/api/microsoft.aspnetcore.mvc.authorization.authorizefilter?view=aspnetcore-3.1) for DI (it has more than a single, default constructor).

Reply:
it's hard to believe there is no other way.
unfortunately I don't have time to dig into it, let it be

Reply:
it's hard to believe there is no other way.
unfortunately I don't have time to dig into it, let it be